### ISSUE TITLE
Dev Env Scripts

### ISSUE THREAD

**Issue by vlbee** [fac-14/DAAMwk5#23]
https://github.com/fac-14/DAAMwk5/blob/master/package.json#L6-L8

I highly recommend setting up some common scripts in your package.json at the start of your project. This ensures you create a consistent dev environment across all developers on the team. 

On your next project try to set up a `test` and `start` script at the start of the process: 

```
  "scripts": {
    "test": "nyc tape src/test.js | tap-spec",
    "start": "nodemon src/server.js",
    "lint": "/node_modules/.bin/eslint"
  },
```

Fyi you can also use nodemon on your test script :)

**Comment by VirtualDOMinic** [comment]
We initially used nodemon on our test script, but @martingaston mentioned it seemed to be causing issues with Travis :(

**Comment by martingaston** [comment]
Yeah - our CI tests were running for 10 minutes and then timing out, is that normal?

**Comment by vlbee** [comment]
Travis uses your default scripts so you can write another one for devs 👍 
```
  "start": "node src/server.js",
   "start:watch": "nodemon src/server.js",
  "test": "nyc tape src/test.js | tap-spec",
  "test:watch": "nodemon nyc tape src/test.js | tap-spec",
```
